main_concept/indexes.md

```diff
@@ -1,17 +1,16 @@
 # Indexes
 
-An `index` is an entity, like a *table in SQL*, with a specific `schema` definition. It gathers a collection of `documents` with the structure defined by the schema. 
+An `index` is an entity, like a *table in SQL*, with a specific `schema` definition. It gathers a collection of `documents` with the structure defined by the schema.
 
 An **index is defined** by an unique identifier `uid` that is generated by MeiliSearch on [index creation](/references/indexes.md#create-an-index). It also has **a name** to help you track your different indexes.
 
 ## Index UID and name
 
-The `UID` of an index is it's unique identifier. It is the `:index` parameter on every `indexes/:index` route. 
+The `uid` of an index is its unique identifier. It is the `:index` parameter on every `indexes/:index` route.
```

Comment:
Same. Don't you think it would be better to change `:index` into `:index-uid`/`:uid`?